e2e_samples/dataset_versioning/datafactory/README.md

```diff
@@ -2,14 +2,22 @@
 
 This folder is for Data pipeline which copy data from SQL DB to Azure Storage (Delta Lake). We are using Azure Data Factory to implement pipeline.
 
+## prerequisites
+Expect you've already finishes previous set-up including..
+- Provision Azure Resources by IaC (Terraform)
+- [Install the Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli)
```

Comment:
Maybe just a small thing.
But actually, we need to install the Azure cli before we can provision the resources with Terraform.